docs/source/tutorials.rst

```diff
@@ -0,0 +1,9 @@
+Tutorials
+=========
+
+.. toctree::
+   :maxdepth: 2
+
+   ../../../tutorial/CP_APR
```

Comment:
Did you test the sphinx build locally? It is broken with the out of tree build here for me. Blocking the PR for the moment. I will provide more details in a a little bit.

Reply:
Whoops, looks like I definitely didn't test that out properly.

Reply:
I'm working on the previous Tutorial PR's where I'll be clearing out the Notebook outputs to help clean things up.

Reply:
No problem! You are helping us to find processes we need to build, or describe better. Ideally, we make it challenging to break things (there are so many other things to keep track of, offloading as much as possible to tools or automation is usually the way to go).